docs/basic-features/image-optimization.md

```diff
@@ -11,199 +11,173 @@ description: Next.js supports built-in image optimization, as well as third part
+**Target the image with className, not based on DOM structure**
 
-### Device Sizes
+The image component has a somewhat more complicated DOM structure than a standard `<img>`. Regardless of layout mode used, the inner `<img>` element will always be wrapped by exactly one `<div>`. It may have a sibling `<div>` for spacing, depending on layout. These additional `<div>`s are what allows us to provide the different layout modes, while ensuring no layout shift.
 
-In some cases, where you know the expected device widths from the users of your website, you can specify a list of device width breakpoints using the `deviceSizes` property. These widths are used when the [`next/image`](/docs/api-reference/next/image.md) component uses `layout="responsive"` or `layout="fill"` so that the correct image is served for the device visiting your website.
+The easiest way to style the inner `<img>` rendered by the image element is to **apply a class to your images using the `className` property** on the Image element and then target that class with your styles. The value of className will be automatically applied to the underlying `<img>` element.
 
-If no configuration is provided, the default below is used.
+**When using `layout='fill'`, the parent element must have `position: relative`**
 
-```js
-module.exports = {
-  images: {
-    deviceSizes: [640, 750, 828, 1080, 1200, 1920, 2048, 3840],
-  },
-}
-```
-
-### Image Sizes
+This is necessary for the proper rendering of the image element in that layout mode.
 
-You can specify a list of image widths using the `imageSizes` property. These widths should be different (usually smaller) than the widths defined in `deviceSizes` because the arrays will be concatenated. These widths are used when the [`next/image`](/docs/api-reference/next/image.md) component uses `layout="fixed"` or `layout="intrinsic"`.
+**When using `layout='intrinsic'`, the parent element must have `display: block`**
 
-If no configuration is provided, the default below is used.
+This is the default for `<div>` elements but should be specified otherwise.
 
-```js
-module.exports = {
-  images: {
-    imageSizes: [16, 32, 48, 64, 96, 128, 256, 384],
-  },
-}
-```
+## Properties
 
-### Minimum Cache TTL
+[**View all properties available to the `next/image` component.**](/docs/api-reference/next/image.md)
 
-You can configure the time to live (TTL) in seconds for cached optimized images. In many cases, its better to use a [Static Image Import](#image-Imports) which will handle hashing file contents and caching the file forever.
+### Styling Examples
 
-```js
-module.exports = {
-  images: {
-    minimumCacheTTL: 60,
-  },
-}
-```
+For examples of the Image Component used with the various fill modes, see the [Image Component Example App](https://image-component.nextjs.gallery/).
```

Comment:
Nit: throughout this doc and the API reference, there are alternate capitalizations of "image component" and "Image Component". Both seem fine with me but we should probably just pick one and stick with it.

Reply:
Good call. I normalized on "Image component" which was being used most often in the API doc, and I think reads best in context.